src/styles/blocks/page.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  display: flex;
+  box-sizing: border-box;
+}
+
+head {
+  display: none;
+}
+
+body {
```

Comment:
The 'body' selector is missing the 'display: flex;' property in order to center the stopwatch properly. Add 'display: flex;' to the 'body' selector.